IonicPortals/src/main/kotlin/io/ionic/portals/PortalsPlugin.kt

```diff
@@ -94,11 +97,7 @@ fun JSONObject.toMap(): Map<String, Any> {
     return map
 }
 
-data class SubscriptionResult(
-    val topic: String,
-    val data: Any,
-    val subscriptionRef: Int
-) {
+data class SubscriptionResult(val topic: String, val data: Any?, val subscriptionRef: Int) {
```

Comment:
The Android Studio formatter automatically did this, I can revert it back if it bothers anyone 🫠.